### DISCUSSION TITLE
Masonry and column layout place all images in single column

### DISCUSSION THREAD

**Discussion by flynnpc-swca** [igordanchenko/react-photo-album#129] [Q&A] [answered]
most if not all of my photos are placed in a single column when I use the column or masonry layout. Rows doesn't even work. I am testing with random Google images, so the height and width are not always known. I accounted for that with a ternary that sets the dimensions to 400x300. Any idea why the layout is not distributing my images accordingly? Thanks.

**Comment by igordanchenko** [comment, marked as answer]
`width` and `height` are required attributes in the `photos` array objects, and the [documentation](https://react-photo-album.com/documentation#Photo) is pretty clear about it.

Please see #120 for possible solutions when photo dimensions aren't known beforehand.
